modules/checking-mco-status.adoc

```diff
@@ -15,11 +15,36 @@ To see the status of the Machine Config Operator, its sub-components, and the re
 ----
 $ oc get machineconfigpool
 NAME      CONFIG                  UPDATED  UPDATING   DEGRADED  MACHINECOUNT  READYMACHINECOUNT  UPDATEDMACHINECOUNT DEGRADEDMACHINECOUNT  AGE
-master    rendered-master-dd…     True     False      False     3             3                  3                                0                     4h42m
-worker    rendered-worker-fde…    True     False      False     3             3                  3                                0                     4h42m
+master    rendered-master-dd…     True     False      False     3             3                  3                                0        4h42m
+worker    rendered-worker-fde…    True     False      False     3             3                  3                                0        4h42m
 ----
 +
+where:
++
+--
+** `UPDATED` indicates that the MCO applied the current machine config to the nodes in that MCP.
+** `UPDATING` indicates that the MCO is applying the current machine config to at least one of the nodes in that MCP. Nodes that are in `UPDATING` status might not available for scheduling.
+** `DEGRADED` indicates the MCO is blocked from applying the current machine config to at least one of the nodes in that MCP, or the configuration is failing. Nodes that are in `DEGRADED` status are not available for scheduling.
+** `MACHINECOUNT` indicates the total number of machines in that MCP.
+** `READYMACHINECOUNT` indicates the total number of machines in that MCP that are updated to the current machine config and are ready for scheduling.
+** `UPDATEDMACHINECOUNT` indicates the total number of machines in that MCP that have the current machine config.
+** `DEGRADEDMACHINECOUNT` indicates the total number of machines in that MCP that are marked as degraded or unreconcilable.
+--
++
 In the previous output, there are three master and three worker nodes. All machines are updated and none are currently updating. Because all nodes are `Updated` and `Ready` and none are `Degraded`, you can tell that there are no issues.
++
+[NOTE]
+====
+If a node is being cordoned, the node is not included in the `READYMACHINECOUNT`, but is included in the `MACHINECOUNT`. Also, the node status is set to `UPDATING`. Because the node has the current machine config, it is counted in the `UPDATEDMACHINECOUNT` total. 
```

Comment:
when the node is cordoned, its status is not `UPDATING`, it should be `Ready, SchedulingDisabled`, mcp status is `UPDATING`, because node is applying desired config.